source/_posts/How-to-add-Lottie-Animation-in-an-Android-app.md

```diff
@@ -0,0 +1,151 @@
+---
+title: How to add Lottie Animation in an Android application
+date: 2020-08-26 18:40:48
+author: Diwakar Singh
+category: Android
+tags:
+- android
+- lottie animation
+- kotlin
+- dynamic feature
+- nayan
+- user interaction
```

Comment:
USER INTERACTION
DYNAMIC FEATURE

USE THIS KEYWORD IN CONTENT ALSO, IT IS NOT SHOWING IN THE CONTENT.